pkg/audit/verifier.go

```diff
@@ -321,7 +321,11 @@ func (verifier *Verifier) Reverify(ctx context.Context, stripe *Stripe) (report
 			limit, err := verifier.orders.CreateAuditOrderLimit(ctx, verifier.auditor, createBucketID(stripe.SegmentPath), pending.NodeID, pending.PieceID, pending.ShareSize)
 			if err != nil {
 				if overlay.ErrNodeDisqualified.Has(err) {
-					verifier.containment.Delete(ctx, piece.NodeId)
+					_, errDelete := verifier.containment.Delete(ctx, piece.NodeId)
+					if errDelete != nil {
+						verifier.log.Debug("Error deleting disqualified node from containment db", zap.Stringer("Node ID", piece.NodeId))
```

Comment:
Add `zap.Error(err)` as last argument to `log.Debug`, so we also see the actual error in the log.